components/member-profile/index.js

```diff
@@ -1,29 +1,34 @@
-import { getDataURL, getImgURL } from 'helper-functions/urls';
-
 import PropTypes from 'prop-types';
-import React from 'react';
 import classNames from './member-profile.module.scss';
-import useFetch from 'custom-hooks/useFetch';
 
 const Profile = (props) => {
-  const imgLink = getImgURL(props.id);
-  const { data } = useFetch(getDataURL(props.id));
+  const {
+    membersData: { first_name, last_name },
+    imageLink
+  } = props;
+  const memberName = `${first_name || ''} ${last_name || ''}`;
   return (
     <>
-      <img src={imgLink} className={classNames.profilePic} alt="profile" />
-      <div className={classNames.memberName}>
-        {data ? `${data['first_name']} ${data['last_name']} ` : `${props.id}`}
-      </div>
+      <img src={imageLink} className={classNames.profilePic} alt="profile" />
+      <div className={classNames.memberName}>{memberName.trim() || '--'}</div>
     </>
```

Comment:
Add `Head` with a title here, otherwise next.js shows the route on the title. 
To see example look [here](https://nextjs.org/docs/api-reference/next/head).
```
<Head>
  <title>{memberName} Profile<title>
</Head>
```

Reply:
@whyDontI has his PR ready with these changes